lib/js-file.js

```diff
@@ -805,7 +827,7 @@ JsFile.prototype = {
  * @param {Object} [esprimaOptions]
  * @returns {Object}
  */
-JsFile.parse = function(source, esprima, esprimaOptions) {
+function parseJavaScriptSource(source, esprima, esprimaOptions) {
```

Comment:
See https://github.com/jscs-dev/node-jscs/issues/1214. It's not private because it's not closed over, not using `@private`, and is not called `_parse`.